generators/csharp/sdk/src/wrapped-request/WrappedRequestGenerator.ts

```diff
@@ -173,12 +173,15 @@ export class WrappedRequestGenerator extends FileGenerator<CSharpFile, SdkCustom
         });
     }
 
-    public doGenerateSnippet(example: ExampleEndpointCall): csharp.CodeBlock {
+    public doGenerateSnippet(example: ExampleEndpointCall, parsedDatetimes: boolean): csharp.CodeBlock {
```

Comment:
Now that we have a `boolean`, it's probably a good time to use an in-lined request. That way it's easier to see what the function is doing at the call-site. Otherwise it's easy to fall into the trap of a long list of anonymous `boolean` positional parameters. We can also name this `parseDatetimes` (re: my other comment).

```typescript
public doGenerateSnippet({
  example,
  parseDatetimes
}: {
  example: ExampleEndpointCall;
  parseDatetimes?: boolean;     // We can make this optional if that's easier, too.
}: csharp.CodeBlock {
  ...
}
```

Then each call-site is adapted to:

```typescript
const snippet = objectGenerator.doGenerateSnippet({
  example: example.shape,
  parseDatetimes: true,
});
```

The same recommendation applies to the `getSnippetForTypeReference` function.

Reply:
yeah great call, changing!